src/spawn.coffee

```diff
@@ -1,33 +1,7 @@
 {spawn} = require 'child_process'
 fs = require 'fs'
 
-killAllProcs = (procs) ->
-  procs.forEach (proc) ->
-    try
-      proc.rawProcess.kill()
-    catch err
-      console.error err.stack
-  procs = []
-
-allProcs = []
-registered = false
-registerUncaughtHandler = (proc) ->
-  allProcs.push(proc)
-
-  if !registered
-    process.on 'uncaughtException', (error) ->
-      killAllProcs(allProcs)
-      throw error
-
-    process.on 'exit', ->
-      killAllProcs(allProcs)
-
-    registered = true
-
-
-procNotFoundError = (error, cmd) ->
-  error.message = "Unable to find #{cmd}"
-  error
+WRAPPER = require.resolve './wrap.js'
```

Comment:
Can we have a comment (perhaps here, maybe just in the README somewhere) that explains why this uses a node wrapper instead of spawning the procs directly?